### ISSUE TITLE
How to redirect the ports

### ISSUE THREAD

**Issue by vmn86** [pi-node/instructions#54]
How do I fix this...... so please help me in layman's language.

![image](https://user-images.githubusercontent.com/78527155/106854789-ec1e1000-66e1-11eb-9ea6-d79f6d44cc60.png)

**Comment by ihatejam** [comment]
Hi - we can help but we need more information. What OS are you using and which version/build are you on? What kind of internet router are you using?
If you are not comfortable with logging into your internet router and configuring it to forward ports or/and not comfortable with firewall adjustments in your PC, then consider using a VPN service that allows port forwarding - Running a VPN will require almost no manual adjustment on your part. (VPN services are usually not free) like Surfshark or NordVPN. 
Pi Network is not affiliated with any VPN service -- If you decide to use one, please test which works best in your environment.

If you want to manually configure things without using a VPN (which is only possible if your ISP allows port forwarding), then follow the instructions contributed by various community members here - https://github.com/pi-node/instructions/wiki (best to avoid older instructions and check out the newer ones instead)
